cmd/jobbuzz-scraper/main.go

```diff
@@ -26,21 +25,25 @@ func main() {
 
 	service := service.Service{DB: db}
 
+	// Scrape JobCenter
+
 	fmt.Println("Fetching jobs from JobCenter")
 	jobs := scraper.ScrapeJobcenter()
 
 	for _, job := range jobs {
 		service.CreateJob(&job)
```

Comment:
err check

Reply:
err check for ScrapeJobCenter?

Reply:
ScrapeJobCenter should return error for the handle error.

I am not sure whether that is possible with go-colly. It's strange that the only error handling sample handles it by printing the error out only. 🤦‍♂️

Maybe doing a panic might be the only way.

Reply:
I think it should be fine to just ignore this for now.